### PR TITLE
feat(article): article about color blindness and links

### PR DESCRIPTION
**Pull request by beingfrankly**

### CODE REVIEW

File: data/blog/are-our-links-visually-accessible.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: 'Are our links visually accessible?'
+date: '2022-08-25'
+tags: ['a11y']
+summary: 'A simple link inside a paragraph, what could go wrong? Well, a lot in fact. Let’s have a closer look, shall we.'
```
Comment (DaveBitter) [inline data/blog/are-our-links-visually-accessible.md]: shall we should end with a question mark right?

File: data/blog/are-our-links-visually-accessible.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: 'Are our links visually accessible?'
+date: '2022-08-25'
+tags: ['a11y']
```
Comment (DaveBitter) [inline data/blog/are-our-links-visually-accessible.md]: Could we perhaps also make an `accessibility` tag?

File: data/blog/are-our-links-visually-accessible.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+summary: 'A simple link inside a paragraph, what could go wrong? Well, a lot in fact. Let’s have a closer look, shall we.'
+authors: ['frank-van-eldijk']
+canonicalUrl: 'https://beingfrankly.nl/blog/are-our-links-visually-accessible'
+---
+
+# Are our links visually accessible?
+
+When I wrote about [the accessibility of links](https://beingfrankly.nl/blog/how-accessible-are-links), I’ve focused on screen reader & voice recognition users. But this time I’ll focus on what visually goes wrong for our links.
```
Comment (DaveBitter) [inline data/blog/are-our-links-visually-accessible.md]: Perhaps nice to link internally on the techhub? so `/articles/how-accessible-are-links`?

File: data/blog/are-our-links-visually-accessible.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+While browsing the web, I keep noticing a design trend more often than before. And that’s the styling of their links.
+
+Having only a color difference to separate them from the rest of the content. You might wonder what’s wrong with only using a color to convey meaning, so let’s find out together.
+
+## Lost in a sea of text
+
+I’ll show you two examples, both showing two paragraphs and a link somewhere in the middle.
+
+![Both examples show the same two paragraphs with a link in the middle. The visual cue for the link is only a difference in color. In the first example it’s orange and in the second example it’s almost dark yellow, which almost looks the same as the rest of the content.](https://storage.googleapis.com/beingfrankly/only_color.png)
```
Comment (DaveBitter) [inline data/blog/are-our-links-visually-accessible.md]: Could you add these to the images folder for this article?

File: data/blog/are-our-links-visually-accessible.md (hunk 1 above)
Comment (DaveBitter) [inline data/blog/are-our-links-visually-accessible.md]: Do you by any chance have a teaser image/hero image?

File: data/blog/are-our-links-visually-accessible.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Do note that the simulated effect might differ from reality. This is due to a lot of factors (e.g. the algorithm used, severity of the color vision deficiency, monitor settings, etc).
+
+Are you able to spot the link in the second example? I bet it wasn’t as easy as the first example. So, what could we do to prevent this from happening?
+
+## More weight please
+
+So, what about giving our links more `font-weight`? Surely that will solve the problem, right?
+
+![Both examples show the same two paragraphs with a link in the middle. The link is now bold and is still a different color than the rest of the text. However, other words inside the paragraphs have been made bold as well.](https://storage.googleapis.com/beingfrankly/bolded.png)
```
Comment (DaveBitter) [inline data/blog/are-our-links-visually-accessible.md]: Could you add these to the images folder for this article?

**Comment by DaveBitter** [comment]
Nice to have another a11y article!

File: data/blog/are-our-links-visually-accessible.md (hunk 1 above)
Comment (beingfrankly) [reply data/blog/are-our-links-visually-accessible.md]: Done

File: data/blog/are-our-links-visually-accessible.md (hunk 2 above)
Comment (beingfrankly) [reply data/blog/are-our-links-visually-accessible.md]: Done

File: data/blog/are-our-links-visually-accessible.md (hunk 4 above)
Comment (beingfrankly) [reply data/blog/are-our-links-visually-accessible.md]: Done

File: data/blog/are-our-links-visually-accessible.md (hunk 5 above)
Comment (beingfrankly) [reply data/blog/are-our-links-visually-accessible.md]: Done

File: data/blog/are-our-links-visually-accessible.md (hunk 3 above)
Comment (beingfrankly) [reply data/blog/are-our-links-visually-accessible.md]: Done

File: data/blog/are-our-links-visually-accessible.md (hunk 1 above)
Comment (beingfrankly) [reply data/blog/are-our-links-visually-accessible.md]: Not yet. But it's in the planning to add hero images for every article :-)

**Review by DaveBitter — APPROVED** [review]

**Pull request merged**